media-source-respec.html

```diff
@@ -2696,6 +2688,284 @@ <h3>Event Summary</h3>
       </section>
     </section>
 
+    <section data-cite="mimesniff" data-dfn-for="ManagedMediaSource">
+      <h2><dfn>ManagedMediaSource</dfn> interface</h2>
+      <p>
+        A {{ManagedMediaSource}} is a {{MediaSource}} that actively manages its
+        memory content. Unlike a {{MediaSource}}, the [=user agent=] can evict content
+        through the [=ManagedMediaSource/memory
+        cleanup=] algorithm from its {{MediaSource/sourceBuffers}} (populated with {{ManagedSourceBuffer}})
+        for any reason.
+      </p>
+<aside class="note" title="Eviction reasons">
+<p>
+  Reasons that the user agent might evict content are implementation specific and can include, but are not limited to, memory and/or hardware limitations, change in environmental conditions, and so on. Developers shouldn't make assumptions as to why, how, or when a user agent might evict content. Instead, developers need to write scripts with the assumption that content is constantly and randomly being evicted to avoid stalled video playback (i.e., code defensibly!). 
```

Comment:
```suggestion
  Reasons that the user agent might evict content are implementation specific and can include, but are not limited to, memory and/or hardware limitations, change in environmental conditions, and so on. Developers shouldn't make assumptions as to why, how, or when a user agent might evict content. Instead, developers need to write scripts with the assumption that content is constantly and randomly being evicted to avoid stalled video playback (i.e., code defensibly and listen for the `bufferedchangeevent`!). 
```